test-files/cases/network/conflicts/highway-015/Expected.osm

```diff
@@ -99,19 +99,22 @@
         <tag k="hoot:id" v="-53980"/>
     </node>
     <way visible="true" id="-54142" timestamp="1970-01-01T00:00:00Z" version="1">
+        <nd ref="-54000"/>
+        <nd ref="-54002"/>
+        <nd ref="-54004"/>
```

Comment:
This change to the output is a gain in sibling way joining making the resulting ways closer to the original in length.